indico/modules/events/papers/templates/management/competences.html

```diff
@@ -0,0 +1,34 @@
+{% from 'forms/_form.html' import form_header, form_field, form_footer %}
+{% extends 'layout/conference_page_base.html' %}
+
+{% block description %}
+    {%- trans -%}
+        Defining competences to the members of the teams may be helpful for the managers and judges during the paper assignment process.
+    {%- endtrans -%}
+{%- endblock %}
+
+{% block content -%}
+    <div id="competences-dialog-content">
+        {{ form_header(form) }}
+            <table class="i-table-widget">
```

Comment:
I'm with @ThiefMaster here. Another point is that this requires understanding of what `form_header` internally does and will most likely confuse anybody reading the code (*"is this indentation intended?"*).